code/game/objects/items/robot/robot_items.dm

```diff
@@ -78,16 +84,21 @@
 					if(M.resting)
 						M.resting = 0
 						M.update_canmove()
+					playsound(loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)

```

Comment:
why type this twice?
it's played no matter if the if or else is checked, just unindent it and leave it at the bottom like it was before